src/gtclang/Driver/Options.inc

```diff
@@ -50,5 +50,6 @@ OPT(bool, ClangFormat, true, "clang-format", "", "Run clang-format on the genera
 OPT(bool, ReportPassPreprocessor, false, "report-pass-preprocessor", "", 
     "Print each line of the preprocessed source prepended by the line number (comments and indentation are removed)", "", false, true)
 OPT(bool, Serialized, false, "loaded-serialized", "", "is the passed file serialized data", "", false, true)
+OPT(bool, WriteToStdOut, false, "write-stdout", "", "write to std out", "", false, true)
```

Comment:
rather than adding a new flag I would do the following: 
If "OutputFile" option is not set, we create an automatic filename <Input>_gen.cpp, which is arbitrary and non intuitive. 
I would simply redo this logic so that if "OutputFile" option is not set, we write to stdout. Otherwise we take the explicit filename specified by options. Like that we dont need an extra "write-stdout" and will be more intuitive.

Reply:
done